src/Core/Checkout/Payment/Cart/Token/TokenStruct.php

```diff
@@ -50,14 +50,20 @@ class TokenStruct extends Struct
      */
     protected $expires;
 
+    /**
+     * @var bool
+     */
+    protected $invalidated;
```

Comment:
please add the native directly on new properties

Reply:
these are both changes done because the bots were complaining 😛 
I reverted/removec the commits

Reply:
what exactly were they complaining about? :thinking:

Reply:
you can run the cs-fixer command, which should fix the code style for you. 

in this case it is not necessary to have the doc type if you already have a native type :wink:

Reply:
https://github.com/shopware/shopware/actions/runs/11212301657/job/31163021677?pr=4952#step:8:64

Reply:
I would suggest, that you only do a property declaration in the constructor. So the following:
```php
    public function __construct(
...
        protected bool $invalidated = false,
    ) {
...
    }